tests/fixtures/docker.py

```diff
@@ -110,26 +112,28 @@ def prefect_base_image(pytestconfig: "pytest.Config", docker: DockerClient):
     return image_name
 
 
-@pytest.fixture(scope="module")
-def registry(docker: DockerClient) -> Generator[str, None, None]:
-    """Starts a Docker registry locally, returning its URL"""
-
-    with silence_docker_warnings():
-        # Clean up any previously-created registry:
+def _wait_for_registry(url: str, timeout: int = 30) -> bool:
```

Comment:
One thing we can rely on here is that we have a global `pytest` timeout, so we don't necessarily need to implement a timeout here

Reply:
Good point!

Reply:
Actually I think we want a timeout here for the messaging in case you haven't started your local registry. Without that you'll just get the pytest timeout and be confused.

Reply:
Ahh that does make sense